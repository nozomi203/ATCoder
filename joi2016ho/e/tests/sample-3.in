15 10
28 1 7
-24 2 1
1 1 1
8 1 1
6 2 1
20 1 3
12 2 2
-10 1 3
7 2 1
5 1 2
