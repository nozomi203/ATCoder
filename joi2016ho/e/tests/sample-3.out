15
14
14
14
14
12
12
12
12
12
12
12
15
15
12
